ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -232,6 +238,15 @@
   @Default("false")
   public final boolean routerUseGetBlobOperationForBlobInfo;
 
+  /**
+   * The custom percentiles of Histogram in operation tracker to be reported. This allows router to emit metrics of
+   * arbitrary percentiles (i.e. 97th, 93th etc). An example of this config is "0.91,0.93,0.97"(comma separated), each
+   * value should fall in {@code [0..1]}.
+   */
+  @Config("router.operation.tracker.custom.percentiles")
+  @Default("")
+  public final List<Double> routerOperationTrackerCustomPercentiles;
```

Comment:
how will these custom percentiles be used by operation tracker in the future? Right now, they are only registered in the `MetricRegistry` for viewing.